Runtime/src/test/java/com/rakuten/tech/mobile/perf/runtime/internal/RuntimeContentProviderSpec.java

```diff
@@ -81,6 +84,18 @@
     verify(TrackerShadow.mockTracker).startMetric("_launch");
   }
 
+  @Test public void shouldStartTrackingForDebugBuildEvenifEnablePercentIsZero() {
+    UtilShadow.mockDebugBuild = true;
+    StoreShadow.cachedContent = new Gson()
+        .fromJson(configZeroPercent.content, ConfigurationResult.class);
+
+    provider.onCreate();
+
+    assertThat(TrackingManager.INSTANCE).isNotNull();
+    verify(TrackerShadow.mockTracker).startMetric("_launch");
+    UtilShadow.mockDebugBuild = false;
```

Comment:
this should probably be part of an `@Before` method.

- [ ] move test setup code to `@Before` method